docs/crio.8.md

```diff
@@ -55,6 +56,8 @@ crio [GLOBAL OPTIONS] config [OPTIONS]
 # GLOBAL OPTIONS
 **--apparmor_profile**="": Name of the apparmor profile to be used as the runtime's default (default: "crio-default")
 
+**--bind-mount-prefix**="": A prefix to use for the source of the bind mounts.
```

Comment:
Lets add more documentation for this.  
```
This option would be useful if you were running CRI-O in a container.  And had `/` mounted on `/host` in your container.  Then if you ran CRI-O with the `--bind-mount-prefix=/host` option, crio would add /host to any bind mounts it is handed over CRI.  If Kubernetes asked to have `/var/lib/foobar` bind mounted into the container, then CRI-I would bind mount `/host/var/lib/foobar`.  Since CRI-O iteself is running in a container with `/` or the host mounted on /hosr, the container would end up with `/var/lib/foobar` from the host mounted in the container rather then `/var/lib/foobar` from the CRIO container.
```

Reply:
> This option would be useful if you were running CRI-O in a container.  And had `/` mounted on `/host` in your container.  Then if you ran CRI-O with the `--bind-mount-prefix=/host` option, crio would add /host to any bind mounts it is handed over CRI.  If Kubernetes asked to have `/var/lib/foobar` bind mounted into the container, then CRI-I would bind mount `/host/var/lib/foobar`.  Since CRI-O iteself is running in a container with `/` or the host mounted on /hosr, the container would end up with `/var/lib/foobar` from the host mounted in the container rather then `/var/lib/foobar` from the CRIO container.

Or maybe something like:

> For example, say you were running CRI-O in a container and had `/` mounted on `/host` in your container.  In that situation, `--bind-mount-prefix=/host` would add `/host` to any CRI-requested bind mounts, making the CRI-O containerization transparent to the CRI consumer.

Reply:
updated